test/e2e/lib/components/site-editor-component.js

```diff
@@ -64,6 +65,25 @@ export default class SiteEditorComponent extends AsyncBaseContainer {
 		);
 	}
 
+	async isBlockInserterOpen() {
+		const inserterMenuLocator = By.css( '.block-editor-inserter__menu' );
+		return await driverHelper.isElementLocated( this.driver, inserterMenuLocator );
+	}
+
+	async openBlockInserter() {
+		const inserterToggleLocator = By.css(
+			'.edit-site-header .edit-site-header-toolbar__inserter-toggle'
+		);
+		await driverHelper.clickWhenClickable( this.driver, inserterToggleLocator );
```

Comment:
Replying to https://github.com/Automattic/wp-calypso/pull/53492/files#r659936044:

I can see what's going on now. Notice that the Inserter opens successfully on the first `openBlockInserter` call and fails only on a subsequent call. It's not because the first click triggers the tooltip, but because in the `openBlockInserter` we're not checking if the Inserter is already open **before** clicking the open/close toggle. TL;DR - the `openBlockInserter` works as a toggle method, so we need to remove the initial click. This should work well and also speed up the spec, as the Inserter will stay open for the subsequent `addBlock` calls:

```suggestion
```

Reply:
([That comment below](https://github.com/Automattic/wp-calypso/pull/53492/files#diff-ac979a382a3361e93e3957848e35d9beaa70d8310bbc5f8d6c64e437fb67b0b1R78-R79) can also be removed!)

Reply:
Gotcha! That seems to do the trick! Pushed a commit